### DISCUSSION TITLE
GitHub Student Developer Pack Approved but Unable to Access Benefits

### DISCUSSION THREAD

**Discussion by SaradaPrasadSwain** [community/community#142310] [GitHub Education]
### Hi everyone,

Hi everyone,

I was approved for the GitHub Student Developer Pack on October 20, but I'm still unable to access any of the benefits. I've checked my account and re-verified my student status, but nothing seems to work. Has anyone else experienced this issue, and if so, how did you resolve it?

Any help or guidance would be appreciated! Thank you.

**Comment by davevad93** [comment]
Hi @SaradaPrasadSwain ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @SaradaPrasadSwain was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by datonefeed** [comment]
response2
